workflow-templates/im-deploy-az-database.yml

```diff
@@ -184,8 +185,11 @@ jobs:
           echo "SQL_USER=$sqlUser" >> $GITHUB_ENV
           echo "SQL_PASSWORD=$sqlPassword" >> $GITHUB_ENV
 
+      # TODO: If your database uses Azure Active Directory Authentication then you can login with a Service Principal
+      #       or Managed Service Identity instead of using the username and password of an account stored in Key Vault.
```

Comment:
Logging in with a managed service identity wouldn't be appropriate from our GitHub action runners. Only a service principal would make sense.